br/pkg/utils/retry.go

```diff
@@ -246,6 +247,8 @@ func WithRetryV2[T any](
 	return *new(T), allErrors // nolint:wrapcheck
 }
 
+var sampleLoggerFactory = logutil.SampleLoggerFactory(time.Minute, 3, zap.String(logutil.LogFieldCategory, "br-utils"))
```

Comment:
```suggestion
var sampleLoggerFactory = logutil.SampleLoggerFactory(time.Minute, 3, zap.String(logutil.LogFieldCategory, "utils"))
```
as lightning will depends on it indirectly, maybe not appear `br`

Reply:
I also want to add something to show it's a "sample logger", so when reading log we don't think the function is really executed for 1 minutes. Do you think `zap.String(logutil.LogFieldCategory, "utils-sample-logger")` is good?

Reply:
maybe `zap.String("sampled", "")`?